config: set names automatically on schema upgrade (#9187)
tarantool/tarantool: changelogs/unreleased/dd-version-id-broadcast.md

## feature/core

> * dd_version_id, which shows the current version of schema, is now broadcasted
>   via box.schema event.

Reviewer: ```suggestion
* `dd_version_id` showing the current schema version is now broadcasted
  via the `box.schema` event.
```
